os/board/esp32_devkit/src/spiram_psram.c

```diff
@@ -531,7 +572,7 @@ esp_err_t IRAM_ATTR psram_enable(psram_cache_mode_t mode, psram_vaddr_mode_t vad
 	}
 
 	WRITE_PERI_REG(GPIO_ENABLE_W1TC_REG, BIT(PSRAM_CLK_IO) | BIT(PSRAM_CS_IO));   //DISABLE OUPUT FOR IO16/17
-	assert(mode < PSRAM_CACHE_MAX && "we don't support any other mode for now.");
+	ASSERT(mode < PSRAM_CACHE_MAX && "we don't support any other mode for now.");
```

Comment:
I think " " sentence is unnecessary. I have no idea for that. Could you let me know the purpose of that?

Reply:
removed, that's for old version assert, useless for ASSERT here. thanks